backend/data/create_africa_geojson.py

```diff
@@ -0,0 +1,104 @@
+These are not committed to the repo because they're too big!
+    https://github.com/nvkelso/natural-earth-vector/blob/master/
+        geojson/ne_50m_admin_0_countries.geojson
+
+    https://github.com/nvkelso/natural-earth-vector/blob/master/
+        geojson/ne_50m_admin_0_breakaway_disputed_areas.geojson
+
+"""
+
+import csv
+import json
+
+
+def load_iso_name_dict():
+    """
+    Read the CSV file from our v-dem based dataset and return
+    a dict mapping ISO A3 codes to country names
+    """
+    csv_name_index = 0
+    csv_iso_code_index = 1
+    iso_name_dict = {}
+    filename = 'lieberman_afr_data.csv'
+    with open(filename, encoding='utf-8') as democracy_score_file:
+        reader = csv.reader(democracy_score_file, delimiter=',')
+        _ = next(reader)  # skip headers
+        for row in reader:
+            iso = row[csv_iso_code_index]
+            name = row[csv_name_index]
+            iso_name_dict[iso] = name
+
+    return iso_name_dict
+
+
+def load_geojson(filename) -> dict:
+    """
+    Read a GeoJSON file from disc
+    and return a dict of the parsed json
+    """
+    with open(filename, encoding='utf-8') as geojson:
+        geojson_string = geojson.read()
+    return json.loads(geojson_string)
+
+
+if __name__ == '__main__':
+    iso_name_dict = load_iso_name_dict()
+
+    all_countries_filename = 'ne_50m_admin_0_countries.geojson'
+    geojson = load_geojson(all_countries_filename)
+
+    disputed_filename = 'ne_50m_admin_0_breakaway_disputed_areas.geojson'
+    disputed_geojson = load_geojson(disputed_filename)
+
+    all_countries_features = geojson['features']
+    disputed_features = disputed_geojson['features']
+
+    africa_features = []
+    africa_iso_codes = iso_name_dict.keys()
+    africa_geojson = {
+        'type': 'FeatureCollection',
+        'features': africa_features,
+    }
+
+    # Construct the new GeoJSON for Africa
+    for country in all_countries_features:
+        properties = country['properties']
+        iso_a3 = properties['ISO_A3']
+        if iso_a3 in africa_iso_codes:
+
+            if iso_a3 == 'ZAF':  # South Africa
+                # This removes the coordinates for Marion Island from South Africa's geometry,
+                coords = country['geometry']['coordinates']
+                coords.pop()
+
+            new_props = {
+                'ISO_A3': iso_a3,
+                'name': iso_name_dict[iso_a3],
+            }
+            country['properties'] = new_props
+            africa_features.append(country)
+    assert(len(africa_features) == 54)  # sanity check
+
+    # sort by name
+    africa_features.sort(key=lambda feature: feature['properties']['name'])
+
+    # Add Western Saraha and Somaliland geometries
+    western_sahara_fid = 18
+    somaliland_fid = 19
+    for territory in disputed_features:
+        fid = territory['properties']['FID']
+        if fid in [western_sahara_fid, somaliland_fid]:
+            africa_features.append(territory)
```

Comment:
Currently, the disputed territories do not have a name or ISO when they are sent to the front end. There are many ways in which we can accomplish that such as copying the jet punk map quiz and letting West Sahara be a part of Morocco and Somaliland be a Somalia. Another way can be to also add in those territories as extra countries. Although Somaliland is in the V-Dem data set, Western Sahara is not.

Reply:
Better to get a temporary solution in now than to merge this as is.

I'm going to combine the geometries for Western Sahara + Morocco and Somaliland + Somalia for now. We can refine with Evan later on.